modules/javafx.media/src/main/native/jfxmedia/Locator/Locator.cpp

```diff
@@ -51,8 +51,28 @@ CLocator::LocatorType CLocator::GetType()
     return m_type;
 }
 
-CLocator::~CLocator()
-{}
+jstring CLocator::LocatorGetStringLocation(JNIEnv *env, jobject locator)
+{
+    static jmethodID mid_toString = 0;
+    jstring result = NULL;
+    CJavaEnvironment javaEnv(env);
+
+    if (mid_toString == 0)
+    {
+        jclass klass = env->GetObjectClass(locator);
+
+        mid_toString = env->GetMethodID(klass, "getStringLocation", "()Ljava/lang/String;");
+        env->DeleteLocalRef(klass);
+        if (javaEnv.clearException())
+            return NULL;
+    }
+
+    result = (jstring)env->CallObjectMethod(locator, mid_toString);
+    if (javaEnv.clearException())
+        return NULL;
+
```

Comment:
A lot of our third-party native code doesn't (I would insist on the `{}` if this were Java code), so if this were a gstreamer file it would be better to follow their convention, but since this is entirely our code, it makes sense to enclose in `{}`.

Reply:
Fixed.